### PR TITLE
add: ajalt/mordant

### LINKED ISSUE #18: Sample wanted for ajalt/mordant
## About the library

- *Description*:    Full-featured text styling for Kotlin command-line applications 
- *GitHub*: https://github.com/ajalt/mordant

## What the sample could do

Print something nice to the console with colors and styles

## Wants to contribute the sample?

- Say so in a comment here
- Follow instructions in [CONTRIBUTING.md](https://github.com/LouisCAD/kotlin-libraries-playground/blob/main/CONTRIBUTING.md)
- Submit your pull request!

### PR DESCRIPTION
**Pull request by DarthBenro008**
Fixes #18 

- changes made as requested in #30 
- added ajalt/mordant

### CODE REVIEW

**Comment by DarthBenro008** [comment]
@jmfayard i have made the requested changes accordingly as mentioned in #30 :)

File: kotlin-jvm/src/main/kotlin/playground/Mordant.kt [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@file:Suppress("PackageDirectoryMismatch")
+
+package playground.mordant
+
+import com.github.ajalt.mordant.TermColors
+
+fun main() {
```
Comment (jmfayard) [inline kotlin-jvm/src/main/kotlin/playground/Mordant.kt:7]: Please add documentation like this, the rest of the PR looks good!

```
/**
 * ajalt/mordant -   Full-featured text styling for Kotlin command-line applications 

- [GitHub](https://github.com/ajalt/mordant)
 */
fun main() {
    println()
    println("ajalt/mordant -   Full-featured text styling for Kotlin command-line applications")
```

Comment (DarthBenro008) [reply kotlin-jvm/src/main/kotlin/playground/Mordant.kt:7]: Yes, sure. will add those lines.

Comment (DarthBenro008) [reply kotlin-jvm/src/main/kotlin/playground/Mordant.kt:12]: @jmfayard added the documentation !

**Comment by jmfayard** [comment]
Thanks @DarthBenro008 !
I merged your changes

**Pull request merged**